src/js/AppDispatcher.js

```diff
@@ -1,3 +1,9 @@
-var Dispatcher = require("flux").Dispatcher;
+var dispatcher = new (require("flux").Dispatcher)();
```

Comment:
@aldipower this should be `Dispatcher`, according to https://github.com/airbnb/javascript#22.8